STEER/STEERBase/AliOADBContainer.h

```diff
@@ -32,22 +44,25 @@ class AliOADBContainer : public TNamed {
   void   RemoveObject(Int_t index);
   void   AddDefaultObject(TObject* obj);
```

Comment:
I think here it must be made clear, preferably with a doxygen comment, that the function AddDefaultObject must get objects which are unique, which are not handled as run-dependent objects, otherwise it ends in a double-delete. From the current interface this is not necessarily clear to the users as has been seen with several containers. It should be stated also in the comment of the member definition. What can go wrong will go wrong, at the worst moment!